goben/client.go

```diff
@@ -53,16 +55,17 @@ func open(app *config) {
 				log.Printf("open: dial %s: %s: %v", proto, hh, errDial)
 				continue
 			}
-			spawnClient(app, &wg, conn, i, app.connections, false)
+			spawnClient(app, &wg, conn, i, app.connections, false, h)
 		}
 	}
 
 	wg.Wait()
 }
 
-func spawnClient(app *config, wg *sync.WaitGroup, conn net.Conn, c, connections int, isTLS bool) {
+func spawnClient(app *config, wg *sync.WaitGroup, conn net.Conn, c, connections int, isTLS bool, targetHost string) {
 	wg.Add(1)
 	go handleConnectionClient(app, wg, conn, c, connections, isTLS)
+	go handleMeasurement(app, targetHost)
```

Comment:
Why not move this call up into open. So you do not have to pass the targetHost variable around.